MAINTAINERS.md

```diff
@@ -1,17 +1,167 @@
+3. MAINTAINERS.md file is updated to represent corresponding areas of responsibility.
+
+## <a name="maintainer"></a>Maintainer
+
+A Maintainer is one of the most established contributors who are responsible for the project technical direction and
+participate in making decisions about strategy and priorities of the project. The responsibilities they have increase
+in scope from that of Code Owners to encompass the entire project. Maintainers may make public presentations and
+statements in public forums (e.g., UXL DevSummit, UXL working group meetings, etc.) about the status and direction of
+the oneDPL project.
+
+Requirements:
+  * Experience as a Code Owner.
+  * Track record of major project contributions to a specific project component.
+  * Demonstrated deep knowledge of a specific project component.
+  * Demonstrated broad knowledge of the project across multiple areas.
+  * Commits to using privileges responsibly for the good of the project.
+  * Is able to exercise judgment for the good of the project, independent of
+    their employer, friends, or team.
+
+Responsibilities in addition to those of Code Owners:
+* Co-own the technical direction of a component or aspect of the library
+* Co-own the project as a whole, including determining strategy and policy for the project
+
+Privileges:
+* Maintain permissions granted to the repository
+* Can participate in promotions of Code Owners and Maintainers
+* Choose project's Milestones during planning
+* Can request rework of RFCs in any part of the project
+* Can manage release process of the project
+* Can represent the project in public as a Maintainer
+
+The process of becoming a Maintainer is:
+1. A Code Owner is nominated by a current Maintainer by opening a PR modifying the MAINTAINERS.md file to move the code owner into the Current Maintainers section.
+2. A majority of the Current Maintainers must approve the PR.
+
+## <a name="leavearole"></a>Leave A Role
+
+Active code owners and maintainers of the oneDPL project may leave their current role by submitting a PR removing their
+name from the list of current maintainers or code owners and tagging two or more active maintainers to review and
+approve the PR.
+
+## <a name="currentcodeowners"></a>Current Code Owners
+
+| Component             | Code Owner(s)       | Github ID | Affiliation |
+| --------------------- | ------------------- | --------- | ----------- |
+| C++ standard policies and CPU backends | Dan Hoeflinger | @danhoeflinger | Intel |
+| SYCL device policies and SYCL backends | Mikhail Dvorskiy<br>Adam Fidel | @MikeDvorskiy<br>@adamfidel |  Intel |
+| Tested Standard C++ APIs & Utility Function Object Classes | Sergey Kopienko | @SergeyKopeinko | Intel |
+| C++17 standard algorithms | Mikhail Dvorskiy<br>Adam Fidel<br>SergeyKopienko<br>Julian Miller<br>Dmitriy Sobolev | @MikeDvorskiy<br>@adamfidel<br>@SergeyKopienko<br>@julianmi<br>@dmitriy-sobolev | Intel<br>Intel<br>Intel<br>Independent<br>Intel<br>Intel |
```

Comment:
Lets remove excessive "Intel" at the last column:

```suggestion
| C++17 standard algorithms | Mikhail Dvorskiy<br>Adam Fidel<br>SergeyKopienko<br>Julian Miller<br>Dmitriy Sobolev | @MikeDvorskiy<br>@adamfidel<br>@SergeyKopienko<br>@julianmi<br>@dmitriy-sobolev | Intel<br>Intel<br>Intel<br>Independent<br>Intel |
```

Reply:
There are non-Intel entries, so I think it's important for Intel not to be an assumed default.

Reply:
No, I simply meant that we need to remove the last "Intel" entry. There are 5 people in that row, but 6 affiliations.

Reply:
Thank you for the clarification. I've got the number correct now.